code/_onclick/hud/new_player.dm

```diff
@@ -437,3 +437,42 @@
 		return
 	var/mob/dead/new_player/new_player = hud.mymob
 	new_player.handle_player_polling()
+
+//NRP MONKE
+/atom/movable/screen/lobby/button/nrp
+	screen_loc = "TOP:-122,CENTER:+164"
+	icon = 'icons/hud/lobby/redirect.dmi'
+	icon_state = "nrp"
+	base_icon_state = "nrp"
+
+/atom/movable/screen/lobby/button/nrp/Click(location, control, params)
+	. = ..()
+	if(!.)
+		return
+	hud.mymob.client << link("byond://198.37.111.92:2102")
```

Comment:
```suggestion
	DIRECT_OUTPUT(hud.mymob.client, link("byond://198.37.111.92:2102"))
```

Reply:
i would love to change that right now but i have to merge it so it can be changed further by aberra